types/ffmpeg.js/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
Why is this needed?

Reply:
I faced with the problem that it is not possible to correctly lint package name if it contains ".". So I turned off this rule. I tried to set a "non-npm package" in the header, but that solution seemed to be improper.
If you have any proposals on how to solve this issue - I'm ready to implement it. Currently, I'm out of ideas.